test-runner/action.yml

```diff
@@ -75,5 +75,11 @@ runs:
       if: always()
       with:
         name: ${{ env.test }} outputs
-        path: ${{ inputs.test_dir }}/logs
+        path: ${{ inputs.test_dir }}/logs/*.log
+        overwrite: true
+    - uses: actions/upload-artifact@v4
+      if: always()
+      with:
+        name: my-artifact ${{ env.test }} json
```

Comment:
Can we use a more descriptive name for the artifact?

Reply:
Sounds good, I have made this `test-runner-summary ${{ env.test }}` instead, and since this is name I will be omitting the json